certification/internal/policy/operator/scc_info.go

```diff
@@ -0,0 +1,102 @@
+package operator
+
+import (
+	"context"
+	"fmt"
+	"os"
+
+	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification/internal/bundle"
+	log "github.com/sirupsen/logrus"
+
+	"github.com/redhat-openshift-ecosystem/openshift-preflight/certification"
+)
+
+// securityContextConstraintsInCSV evaluates the csv and logs a message if a non default security context constraint is
+// needed by the operator
+type securityContextConstraintsInCSV struct {
+	customSCCSpecified bool
+}
+
+func NewSecurityContextConstraintsCheck() *securityContextConstraintsInCSV {
+	return &securityContextConstraintsInCSV{
+		false,
+	}
+}
+
+func (p *securityContextConstraintsInCSV) Validate(ctx context.Context, bundleRef certification.ImageReference) (bool, error) {
+	requestedSCCList, err := p.dataToValidate(ctx, bundleRef.ImageFSPath)
+	if err != nil {
+		return false, err
+	}
+
+	return p.validate(ctx, requestedSCCList)
+}
+
+func (p *securityContextConstraintsInCSV) dataToValidate(ctx context.Context, imagePath string) ([]string, error) {
+	csvFilepath, err := bundle.GetCsvFilePathFromBundle(imagePath)
+	if err != nil {
+		return nil, err
+	}
+
+	csvFileReader, err := os.Open(csvFilepath)
+	if err != nil {
+		return nil, err
+	}
+
+	requestedSccList, err := bundle.GetSecurityContextConstraints(ctx, csvFileReader)
+	if err != nil {
+		return nil, fmt.Errorf("unable to extract security context constraints from ClusterServiceVersion: %w", err)
+	}
+
+	return requestedSccList, nil
+}
+
+func (p *securityContextConstraintsInCSV) validate(ctx context.Context, requestedSccList []string) (bool, error) {
+	if len(requestedSccList) == 0 {
+		log.Infof("No custom security context constraint was detected in the CSV. The default the restricted SCC will be used.")
```

Comment:
There is still an extra 'the' in the second sentence.

Reply:
oops sorry typo blindness